sys/boot/common/console.c

```diff
@@ -98,17 +107,14 @@ getchar(void)
 {
     int		cons;
     int		rv;
-    
+
     /* Loop forever polling all active consoles */
-    for(;;) {
-	for (cons = 0; consoles[cons] != NULL; cons++) {
-	    if ((consoles[cons]->c_flags & C_ACTIVEIN) && 
-		((rv = consoles[cons]->c_in()) != -1)
-	    ) {
+    for(;;)
+	for (cons = 0; consoles[cons] != NULL; cons++)
+	    if ((consoles[cons]->c_flags & (C_PRESENTIN | C_ACTIVEIN)) ==
+		(C_PRESENTIN | C_ACTIVEIN) &&
+		((rv = consoles[cons]->c_in()) != -1))
 		return(rv);
-	    }
-	}
-    }
 }
```

Comment:
These braces should be kept.  If the LoC >= 2, better to use braces explicitly.

Reply:
Thanks ! (To be honest i actually like explicit branches :-))